examples/gallery/lines/linestrings.py

```diff
@@ -0,0 +1,50 @@
+# ruff: noqa: RUF003
```

Comment:
```suggestion
```

Reply:
@michaelgrund seems you applied @seisman's suggestion in commit https://github.com/GenericMappingTools/pygmt/pull/3711/commits/47dabbc540fa009172ac92da4469b5c3b6e4d8ca, but reverted it (accidently ?) in commit https://github.com/GenericMappingTools/pygmt/pull/3711/commits/1dbebb57d5c1345369cef7fb11668edea0aa0193 🙈.
```suggestion

```